packages/analytics/src/integrations/Omnitracking/definitions.ts

```diff
@@ -439,9 +439,16 @@ export const trackEventsMapper: Readonly<OmnitrackingTrackEventsMapper> = {
   },
   [EventTypes.CHECKOUT_STARTED]: data => ({
     tid: 2918,
-    basketValue: data.properties?.total,
+    ...getCheckoutEventGenericProperties(data),
     basketCurrency: data.properties?.currency,
+    basketValue: data.properties?.total,
+    checkoutStep: data.properties?.step,
+    deliveryType: data.properties?.deliveryType,
     lineItems: getProductLineItems(data),
+    paymentType: data.properties?.paymentType,
+    shipping: data.properties?.shipping,
+    shippingTier: data.properties?.shippingTier,
+    tax: data.properties?.orderVAT,
```

Comment:
Isn't this the wrong way around, like this?

```suggestion
    orderVAT: data.properties?.tax,
```

Reply:
Fixed it! Also updated the test snapshot